src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -8,7 +8,31 @@
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws CustomException {
+        if (user.getAge() == null || user.getPassword() == null || user.getLogin() == null) {
+            throw new CustomException("User login,password or age is null!"
+                    + user.getAge() + user.getPassword() + user.getLogin());
+        }
+        if (user.getLogin().isEmpty() || (user.getPassword().isEmpty())) {
+            throw new CustomException("User login or password is empty!"
+                    + user.getLogin() + user.getPassword());
```

Comment:
Similar to the previous comment, avoid concatenating sensitive information like login and password in exception messages. It's better to keep the message generic for security reasons.